### DISCUSSION TITLE
docker pull EOF

### DISCUSSION THREAD

**Discussion by kimyu-ng** [moby/moby#48396] [Q&A]
How would I be able to figure out which port required during docker pull aside regular 443?

`docker pull {{accountId}}.dkr.ecr.us-{{region}}.amazonaws.com/{{repo}}:{{tag}}`

Is there a way to capture such traffic or getting into verbose mode?

```
c6a83fedfae6: Downloading
c6a83fedfae6: Retrying in 5 seconds
1a7b6d7697ab: Retrying in 5 seconds
3f88c649e5b6: Downloading
3f88c649e5b6: Retrying in 5 seconds
c6a83fedfae6: Retrying in 4 seconds
1a7b6d7697ab: Retrying in 4 seconds
3f88c649e5b6: Retrying in 4 seconds
c6a83fedfae6: Retrying in 3 seconds
1a7b6d7697ab: Retrying in 3 seconds
3f88c649e5b6: Retrying in 3 seconds
c6a83fedfae6: Retrying in 2 seconds
1a7b6d7697ab: Retrying in 2 seconds
3f88c649e5b6: Retrying in 2 seconds
c6a83fedfae6: Retrying in 1 second
1a7b6d7697ab: Retrying in 1 second
3f88c649e5b6: Retrying in 1 second
error pulling image configuration: download failed after attempts=6: EOF
```

**Comment by thaJeztah** [comment]
Not sure if other ports are required, but it's possible that other domains are required to be accessible. When pulling the actual content, the registry may return a redirect URL from which the content must be pulled.

That said, the docker daemon logs may provide more information about requests being made (if possible with debug enabled)

also cc @austinvazquez in case you know of relevant reference material

**Reply by kimyu-ng** [reply]
Thanks for prompt reply, how would i be able to determine which domains required in this pull process especially there is no verbose mode? 

Where can I obtain the logs?
